hypertrace-core-graphql-common-schema/gradle.lockfile

```diff
@@ -0,0 +1,59 @@
+# This is a Gradle generated file for dependency locking.
+# Manual edits can break the build and are not advised.
+# This file is expected to be part of source control.
+aopalliance:aopalliance:1.0=compileClasspath,runtimeClasspath
+com.auth0:java-jwt:4.4.0=runtimeClasspath
+com.auth0:jwks-rsa:0.22.0=runtimeClasspath
+com.fasterxml.jackson.core:jackson-annotations:2.15.2=compileClasspath,runtimeClasspath
+com.fasterxml.jackson.core:jackson-core:2.15.2=compileClasspath,runtimeClasspath
+com.fasterxml.jackson.core:jackson-databind:2.15.2=compileClasspath,runtimeClasspath
+com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.15.2=compileClasspath,runtimeClasspath
+com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.15.2=runtimeClasspath
+com.fasterxml.jackson:jackson-bom:2.15.2=compileClasspath,runtimeClasspath
+com.google.android:annotations:4.1.1.4=runtimeClasspath
+com.google.api.grpc:proto-google-common-protos:2.17.0=runtimeClasspath
+com.google.code.findbugs:jsr305:3.0.2=compileClasspath,runtimeClasspath
+com.google.code.gson:gson:2.10.1=runtimeClasspath
+com.google.errorprone:error_prone_annotations:2.18.0=compileClasspath,runtimeClasspath
+com.google.guava:failureaccess:1.0.1=compileClasspath,runtimeClasspath
+com.google.guava:guava-parent:32.1.2-jre=compileClasspath,runtimeClasspath
+com.google.guava:guava:32.1.2-jre=compileClasspath,runtimeClasspath
+com.google.guava:listenablefuture:9999.0-empty-to-avoid-conflict-with-guava=compileClasspath,runtimeClasspath
+com.google.inject:guice:6.0.0=compileClasspath,runtimeClasspath
+com.google.j2objc:j2objc-annotations:2.8=compileClasspath
+com.google.protobuf:protobuf-java:3.24.1=runtimeClasspath
+com.graphql-java-kickstart:graphql-java-kickstart:14.0.0=compileClasspath,runtimeClasspath
+com.graphql-java-kickstart:graphql-java-servlet:14.0.0=compileClasspath,runtimeClasspath
+com.graphql-java:graphql-java-extended-scalars:17.0=compileClasspath,runtimeClasspath
+com.graphql-java:graphql-java:19.6=compileClasspath,runtimeClasspath
+com.graphql-java:java-dataloader:3.2.0=compileClasspath,runtimeClasspath
+io.github.graphql-java:graphql-java-annotations:9.1=compileClasspath,runtimeClasspath
+io.grpc:grpc-api:1.57.2=runtimeClasspath
+io.grpc:grpc-bom:1.57.2=compileClasspath,runtimeClasspath
+io.grpc:grpc-context:1.57.2=runtimeClasspath
+io.grpc:grpc-core:1.57.2=runtimeClasspath
```

Comment:
This could be tangential but does this grpc version have the fix for RST stream vulnerability?

Reply:
transport level concern, but netty does as of 100:
https://netty.io/news/2023/10/10/4-1-100-Final.html

we constrain to at least that version via the BOM (see 
hypertrace-core-graphql-service/gradle.lockfile lines 55-66)